Csharp/README.md

```diff
@@ -0,0 +1,19 @@
+# .NET Core IoT implementation for Dexter Industries Sensors
+
+You will find a full implantation of  Dexter Industries Sensors on [.NET Core IoT repository](https://github.com/dotnet/iot). You will be able to program GoPiGo using C#. The full documentation and example is located into the main repository.
```

Comment:
the sensors are also available for the BrickPi and the GrovePi.

Reply:
Fully agree! Updated